app/views/incomes/editPension.scala.html

```diff
@@ -19,70 +19,69 @@
-                            <th class="text--right" id="amountLabel">@messages("tai.incomes.amount.new")</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr class="income-data">
-                            <td class="break-word">@income("description").value @income("name").value</td>
-                            @income.errors(income("newAmount").name).map{ error =>
-                                <span class="error-notification">@messages(error.message)</span>
-                            }
-                            <td class="text--right"><input type="hidden" id="oldAmount" name='@income("oldAmount").name' value='@income("oldAmount").value' />@income("oldAmount").value</td>
-                            <td class="text--right"><input aria-labelledby="amountLabel" class="text--right input--small income-data-input" type="text" id="newAmount" name='@income("newAmount").name' value='' /></td>
-                        </tr>
-                    </tbody>
-                </table>
-                <input type="hidden" id="name" name='@income("name").name' value='@income("name").value' />
-                <input type="hidden" id="worksNumber" name='@income("worksNumber").name' value='@income("worksNumber").value' />
-                <input type="hidden" id="jobTitle" name='@income("jobTitle").name' value='@income("jobTitle").value' />
-                <input type="hidden" id="startDate" name='@income("startDate").name' value='@income("startDate").value' />
-                <input type="hidden" id="endDate" name='@income("endDate").name' value='@income("endDate").value' />
-                <input type="hidden" id="description" name='@income("description").name' value='@income("description").value' />
-                <input type="hidden" id="employmentId" name='@income("employmentId").name' value='@income("employmentId").value' />
-                <input type="hidden" id="hasMultipleIncomes" name='@income("hasMultipleIncomes").name' value='@income("hasMultipleIncomes").value' />
-            </div>
+    @form(action = controllers.routes.IncomeController.editPensionIncome) {
+    <input type="hidden" name="pensionToDate" value="@pensionToDate">
+    <div class="form-field-group @if(income.hasErrors) {error} margin-bottom-large">
 
-            <div class="form-field-group">
-                <p><input type="submit" class="button" value='@messages("tai.next")'/></p>
+        <div class="grid-row">
+            @income.errors(income("newAmount").name).map{ error =>
+            <span class="error-notification">@messages(error.message)</span>
+            }
+            <div class="column-half">
+                <label class="form-label form-label-bold">@messages("tai.incomes.amount.current.pension")</label>
+                <input type="hidden" id="oldAmount" name='@income("oldAmount").name' value='@income("oldAmount").value' />£@income("oldAmount").value
```

Comment:
How does this handle responsibly? There should be a gap between the old and new when stacked